getstation/website-v3
# [APP-372] Feature/delete user

My actual `.env.development` and `.env.production` file look like:
```
# Prismic
API_KEY=[KEY]

# Auth0
AUTH0_DOMAIN=station.eu.auth0.com
AUTH0_CLIENT_ID=[ID]

# GraphQL
GRAPHQL_URI=http://127.0.0.1:4001/graphql

```

## Test instructions
first, you need to start station api locally on branch `feature/delete-user`
please see this PR for more infos: https://github.com/getstation/api/pull/42

#### development
- yarn develop

#### bundled app
- cp .env.development .env.production
- yarn build
- yarn serve
